### ISSUE TITLE
Incorrect datetime in pybats.bats Bats2d attrs

### ISSUE THREAD

**Issue by evandegriff** [spacepy/spacepy#543]
My issue is ... when opening SWMF output files using Bats2d, the 'time' attribute does not match the datetime of the filename

### Minimal example to reproduce issue:

```
import spacepy.pybats.bats as pbs
output_file = pbs.Bats2d(something.out)
output_file.attrs['time']
```

### OS, Python version, and dependency version information:

```
Darwin-18.7.0-x86_64-i386-64bit
sys.version_info(major=3, minor=7, micro=10, releaselevel='final', serial=0)
numpy=1.20.3
scipy=1.6.3
matplotlib=3.4.2
```

### Version of SpacePy

I am using the latest git version

**Comment by spacecataz** [comment]
More info: this happens with any file whose run time is not zero.  Run time is added to the start date/time of the file to accommodate *.outs files that have many internal runtime values.  However, an adjustment isn't made if the first run time of the file is greater than zero.

Easy fix; will get it this weekend.

**Comment by jtniehof** [comment]
Closed in #558.
